<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResizeDialog</class>
 <widget class="QDialog" name="ResizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dimensions du diaporama</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="automaticSize">
       <property name="text">
        <string>Utiliser la taille de l'écran :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="screenList"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="customSize">
       <property name="text">
        <string>Personalisé :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="width">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="height">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="lockRatio">
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResizeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResizeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>width</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ResizeDialog</receiver>
   <slot>widthChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>height</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ResizeDialog</receiver>
   <slot>heightChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lockRatio</sender>
   <signal>toggled(bool)</signal>
   <receiver>ResizeDialog</receiver>
   <slot>lockRatioToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>screenList</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ResizeDialog</receiver>
   <slot>selectedScreenChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>widthChanged(int)</slot>
  <slot>heightChanged(int)</slot>
  <slot>lockRatioToggled(bool)</slot>
  <slot>selectedScreenChanged(int)</slot>
 </slots>
</ui>
